earth2mip/filesystem.py

```diff
@@ -96,10 +96,10 @@ def _to_url(scheme, path):
         return path
 
 
-def glob(pattern: str) -> List[str]:
+def glob(pattern: str, maxdepth=1) -> List[str]:
```

Comment:
Simple update to glob just so people can give more options for H5 files. Thus supporting a single location or the internal (train/test/outofsample) setup.